CMSgov/design-system
# [WNMGGDS-396] Create linting scripts

### Summary
- Moved stylelint gulp task to new packages
- Added eslint gulp task to new packages (replaces cli command that was previously used)
- Added a `--fix` option
- Removed old eslint and stylelint config packages

### How to test
- Make some linter errors in JS & CSS files in the source and docs directories
   - Easy CSS linter error is to remove new lines after each class definition
   - Easy JS linter error is to change a `const` to a `let`
- Run `yarn lint` and ensure all errors are reported
- Run `NODE_ENV=test yarn lint` and ensure the command errors out
- Run `NODE_ENV=test yarn lint --fix` and ensure the errors are automatically fixed and the command doesn't error

Child DS
- Download [child-design-system-example.zip](https://github.com/CMSgov/design-system/files/4519040/child-design-system-example.zip)
) and extract to same directory as the `design-system` repo
- Run through the same tests as above ^^ (there will be more errors because I didnt bother fixing them)
